Library/Formula/iproute2mac.rb

```diff
@@ -0,0 +1,18 @@
+class Iproute2mac < Formula
+  desc "CLI wrapper for basic network utilities on Mac OS X - ip command"
+  homepage "https://github.com/brona/iproute2mac"
+  url "http://github.com/brona/iproute2mac/archive/v1.1.0.tar.gz"
+  sha256 "3149dc23d5487e837646f50572d512a31799c74cf21288609887cbf8e465d4e6"
+
+  depends_on :python
```

Comment:
Added if `MacOS.version <= :snow_leopard` in  0a71f28 according to https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Python-for-Formula-Authors.md#python-declarations

Is there anything else I can do to help push this forward?